server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -638,6 +638,43 @@ public void run() {
    */
   private Map<DataLevel,Set<KeyExtent>> partitionMigrations() {
     final Map<DataLevel,Set<KeyExtent>> partitionedMigrations = new EnumMap<>(DataLevel.class);
+
+    // Don't try to check migrations if the Root and Metadata
+    // tables are not hosted.
+    boolean skipMigrationCheck = false;
```

Comment:
I'm thinking we want to do this check everywhere we scan for migrations. This is done in several places in Manager.

Reply:
Would be better to structure the code such that it ok if the thread gets stuck trying to read migrations.